scripts/configure.mjs

```diff
@@ -391,6 +393,26 @@ export const getConfig = (() => {
             return getPlatformPackage("react-native-macos", targetVersion);
           },
         },
+        visionos: {
+          files: {
+            Podfile: podfileVisionOS(name, testAppRelPath),
+          },
+          oldFiles: [
+            "Podfile.lock",
+            "Pods",
+            `${name}.xcodeproj`,
+            `${name}.xcworkspace`,
+          ],
+          scripts: {
+            "build:visionos":
+              "mkdirp dist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.visionos.jsbundle --assets-dest dist",
+            visionos: `react-native run-visionos${projectPathFlag}`,
```

Comment:
I am seeing a failure of `run-visionos` not existing. This makes sense because React Native macOS doesn't have that command. I think all these commands are going to the CLI though, so I don't want to add one to React Native macOS. Furthermore, now that `@callstack/react-native-visionos` has a 0.73 version, I can switch this PR to use the callstack fork as the default.